src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BaseTemplatedView.shared.cs

```diff
@@ -32,7 +32,7 @@ protected override void OnChildAdded(Element child)
 			if (control == null && child is TControl content)
 			{
 				control = content;
-				OnControlInitialized(Control);
+				OnControlInitialized(control);
```

Comment:
```suggestion
				OnControlInitialized(Control);
```

Reply:
Control is nullable, we will have to check it for null.
Let's leave as is :)

Reply:
but in line, before you do `control = content`, so in this case, the property will not be null, right? and also on line 32 should be `Control == null` (?)

Reply:
Yes, property won't be null, but nullable checker is not so smart, I believe)

Reply:
As for line 32, yes it can be "Control". But it's the same, right?)

Reply:
Yeah, but just to keep the pattern to use the prop. everywhere. But if I'm being pepega just ignore me

Reply:
No worreis.
Updated!